go/interpreter/lfvm/instructions_test.go

```diff
@@ -1380,6 +1380,69 @@ func TestInstructions_EIP2929_SSTOREReportsOutOfGas(t *testing.T) {
 	}
 }
 
+func TestInstructions_StorageOps_CallStorageContext(t *testing.T) {
+	address := tosca.Address{}
+	_, _ = rand.Read(address[:])
```

Comment:
as I commented in https://github.com/Fantom-foundation/Tosca/pull/860#discussion_r1798857499 
this is a repeated pattern that could be a util func

Reply:
discussed in 860